hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -839,54 +954,103 @@
       tests: []
     }
 
-    // CSRNG tests:
+    // ENTROPY_SRC (pre-verified IP) integration tests:
     {
-      name: chip_csrng_cmd
-      desc: '''Verify the cmd interface to CSRNG.
+      name: chip_entropy_src_ast_rng_req
+      desc: '''Verify the RNG req to ast.
 
-            Details TBD. SW test validates the reception of cmd req done interrupt.
+            Details TBD.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_csrng_entropy_src
-      desc: '''Verify the interface to entropy_src.
+      name: chip_entropy_src_ast_fips
+      desc: '''Verify the connectivity of rng_fips_o feedback signal to RNG.
 
             Details TBD.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_csrng_fuse
-      desc: '''Verify the fuse input to CSRNG.
+      name: chip_entropy_src_csrng
+      desc: '''Verify the transfer of entropy bits to CSRNG.
 
             Details TBD.
             '''
       milestone: V2
       tests: []
     }
-
-    // ENTROPY_SRC (pre-verified IP) integration tests:
     {
-      name: chip_entropy_src_ast_rng_req
-      desc: '''Verify the RNG req to ast.
+      name: chip_entropy_src_cs_aes_halt
+      desc: '''Verify the aes halt handshake with CSRNG.
 
-            Details TBD. SW test validates the reception of the entropy valid interrupt.
+            Details TBD.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_entropy_src_fuse
+      name: chip_entropy_src_fuse_en_fw_read
       desc: '''Verify the fuse input entropy_src.
 
             Details TBD.
             '''
       milestone: V2
       tests: []
     }
+    {
+      name: chip_entropy_src_xht
+      desc: '''Verify the entropy_src extended health test interface.
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // CSRNG tests:
+    {
+      name: chip_csrng_edn_cmd
+      desc: '''Verify incoming command interface from EDN.
+
+            Verify that there are no misconnects between each EDN requesting RNG bits from CSRNG.
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_csrng_fuse_en_sw_app_read
+      desc: '''Verify the fuse input to CSRNG.
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_csrng_lc_hw_debug_en
+      desc: '''Verify the effect of LC HW debug enable on CSRNG.
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // EDN (pre-verified IP) integration tests:
+    {
+      name: chip_edn_entropy_reqs
+      desc: '''Verify the entropy requests from all peripherals.
+
+            Verify that there are no misconnects between each peripheral requesting entropy.
+            X'ref'ed with each IP test that requsts entropy from EDN.
```

Comment:
This test maybe should be broadened to show a boot to post-boot load, then cycle all entropy complex blocks off, then enable again at the full chip level. This will also show any dead lock conditions, such as the RNG AST block might have since it both sources the raw entropy and sinks processed EDN entropy.